setup.py

```diff
@@ -1,9 +1,12 @@
 from setuptools import setup, find_packages
 
 
-version = '0.1.2'
+version = '0.1.3'
 
-requires = []
+requires = [
+    'pysaml2>=1.0.0',
+    'redis>=2.10.5',
```

Comment:
As to being scoped for web apps, I'm not treating it that way, I'm assuming that it should contain whatever may be commonly used by the microservices and the pyramid apps.